src/main/java/ua/chernonog/onlinebookstore/exception/dto/ErrorDto.java

```diff
@@ -0,0 +1,12 @@
+package ua.chernonog.onlinebookstore.exception.dto;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+
+public record ErrorDto(
```

Comment:
```suggestion
public record ErrorRespone(
```
TBH I like it more, but up to you

Reply:
Agree